node/impl/full/eth.go

```diff
@@ -1270,6 +1307,35 @@ func (e *EthEventHandler) EthGetLogs(ctx context.Context, filterSpec *ethtypes.E
 	return ethFilterResultFromEvents(ctx, ces, e.SubManager.StateAPI)
 }
 
+func (e *EthEventHandler) waitForHeightProcessed(ctx context.Context, height abi.ChainEpoch) error {
```

Comment:
do you think it's safe enough to just rely on the RPC call timeout to time this out rather than having an explicit one in here?

Reply:
Have added a 3 epohc timeout here just in case.

Reply:
Done.